eng/pipelines/rpaas-private-check.yml

```
@@ -0,0 +1,43 @@
+trigger: none
+pr:
+  paths:
+    include:
+      # Trigger for files that will result in a specification change
+      - specification/**
```

```suggestion
      - specification/**/resource-manager/**
```

Could we limit to only PRs impacting swaggers under the resource-manager folder?